lib/roken/version-script.mapin

```diff
@@ -229,13 +309,9 @@ HEIMDAL_ROKEN_2.0 {
 		unparse_time_approx;
 		unparse_units;
 		unparse_units_approx;
-		verr;
-		verrx;
-		vwarn;
-		vwarnx;
-		warn;
-		warnx;
-		writev;
+#ifndef HAVE_WRITEV
```

Comment:
We don't even build `lib/roken/writev.c` on non-Windows.

Reply:
For the sake of keeping this PR focused, this should be fine to keep in here right? We can remove `writev.c` in a separate commit?

Reply:
Just wrapping the `writev` symbol with `#ifndef HAVE_WRITEV` should suffice, though on Unix systems it will always be that we `HAVE_WRITEV`, so you can also just remove it from `lib/roken/version-script.mapin`.

Reply:
Thanks, removing it then.